hs_core/views/__init__.py

```diff
@@ -458,10 +458,10 @@ class FilterForm(forms.Form):
 
 @processor_for('my-resources')
 def my_resources(request, page):
-    # import sys
-    # sys.path.append("/home/docker/pycharm-debug")
-    # import pydevd
-    # pydevd.settrace('172.17.42.1', port=21000, suspend=False)
+    import sys
```

Comment:
@selimnairb Good catch. @shawncrawley @zhiyuli Please keep the original commented debug lines without uncommenting them; otherwise, it will break the deployment.